app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,7 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.all
+  end
 
   def show
     # @postId = Post.find(params[:id])
```

Comment:
- [ ] Great job working on this milestone. Kindly remove any unused or commented pieces of code from your codebase. This makes your codebase clean and profesional.